modules/KalturaSupport/components/dualScreen/imagePlayer.js

```diff
@@ -108,6 +123,7 @@
 
 			this.bind( 'onplay', function () {
 				_this.loadAdditionalAssets();
+				_this.syncByReachedCuePoints();
```

Comment:
@esakal what happens if you seek while paused? playback state is retained on seek, e.g. if you were playing and preformed a seek you would keep playing after seek, if you were seeked while paused seek will happen but you would still be paused afterwards.